portfolio/src/main/webapp/script.js

```diff
@@ -99,13 +99,240 @@ function getComments() {
+      {
+        featureType: "road",
+        stylers: [
+          { visibility: "off" }
+        ]
+      }
+    ]
+  });
+
+  fetch("/documents/countries.json").then(resp => resp.json()).then(countriesJsonFile => {
+
+    let countries = new Countries(countriesJsonFile);
+    let randCountryCodeStr = createRandCountryDOM(countries);
+    let incorrectMarkersArray = [];
+
+    document.getElementById("skip_btn").addEventListener("click", function() {
+
+      if (countries.empty()) {
+        deleteElementContentsById("rand_country");
+        deleteAnswerDOM();
+        deleteElementContentsById("selected_country");
+        noCountriesLeftDOM();
+
+        return;
+      }
+
+      deleteElementContentsById("rand_country");
+      randCountryCodeStr = createRandCountryDOM(countries);
+    });
+
+    map.addListener('click', function(mapMouseEvent) {
+
+      if (countries.empty()) {
+        deleteElementContentsById("rand_country");
+        deleteAnswerDOM();
+        deleteElementContentsById("selected_country");
+        noCountriesLeftDOM();
+
+        return;
+      }
+
+      const geocoder = new google.maps.Geocoder;
+
+      geocoder.geocode({ 'location': mapMouseEvent.latLng }, function(address, status) {
+
+        if (status === "OK") {
+
+          deleteAnswerDOM();
+          deleteElementContentsById("selected_country");
+
+          const addressArray = Array.from(address);
+          const addressComps = addressArray[addressArray.length - 1].address_components
+
+          let countryCodeStr;
+          let countryNameStr;
+
+          for (let comp of addressComps) {
+
+            if (comp.short_name.length === 2) {
+              countryCodeStr = comp.short_name;
+              countryNameStr = comp.long_name;
+              break;
+            }
+            if (comp.short_name.includes("Ocean") || comp.long_name.includes("Ocean")) {
+              countryCodeStr = comp.short_name;
+              countryNameStr = comp.long_name;
+            }
+          }
+
+          selectedCountryDOM(countryNameStr);
+
+          if (countryCodeStr === randCountryCodeStr) {
+            correctAnswerDOM(true);
+            deleteElementContentsById("rand_country");
+            countries.removeCountry(randCountryCodeStr);
+            randCountryCodeStr = createRandCountryDOM(countries);
+
+            var checkmark_marker = new google.maps.Marker({
+              position: mapMouseEvent.latLng,
+              map: map,
+              icon: "images/checkmark_icon.png",
+              animation: google.maps.Animation.DROP,
+            });
+
+            // Remove Incorrect Markers
+            incorrectMarkersArray.forEach(mrk => {
+              mrk.setMap(null);
+            });
+          }
+          else {
+            correctAnswerDOM(false);
+
+            var cross_mark_marker = new google.maps.Marker({
+              position: mapMouseEvent.latLng,
+              map: map,
+              icon: "images/cross_mark_icon.png",
```

Comment:
It looks like this is the only property that's actually different on this marker vs. the one in the `if` case. Could you consolidate them (maybe with a helper function)?